Doc/library/math.rst

```diff
@@ -207,6 +207,19 @@ Number-theoretic and representation functions
    of *x* and are floats.
 
 
+.. function:: perm(n, k)
+
+   Return the distinct number of ways to choose *k* items from *n* items
```

Comment:
"distinct number of ways" doesn't quite make sense. "number of distinct ways"? Or just lose the "distinct" word altogether.